test/integration/targets/pids/tasks/files/obtainpid.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
This should be in `test/integration/targets/pids/files/obtainpid.sh` instead, since it's convention for Ansible roles to have a `files` directory at the same level as the `tasks` directory:

https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-directory-structure